_includes/payment-order-other-features.md

```diff
@@ -0,0 +1,1302 @@
+        "id": "/psp/paymentorders/{{ page.payment_order_id }}",
+        "created": "2018-09-14T13:21:29.3182115Z",
+        "updated": "2018-09-14T13:21:57.6627579Z",
+        "operation": "Purchase",
+        "state": "Ready",
+        "currency": "SEK",
+        "amount": 1500,
+        "vatAmount": 0,
+        "remainingCaptureAmount": 1500,
+        "remainingCancellationAmount": 1500,
+        "remainingReversalAmount": 0,
+        "description": "Test Purchase",
+        "initiatingSystemUserAgent": "PostmanRuntime/3.0.1",
+        "userAgent": "Mozilla/5.0...",
+        "language": "nb-NO",
+        "urls" : { "id": "/psp/paymentorders/{{ page.payment_order_id }}/urls" },
+        "payeeInfo" : { "id": "/psp/paymentorders/{{ page.payment_order_id }}/payeeinfo" },
+        "settings": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/settings" },
+        "payers": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/payers" },
+        "orderItems" : { "id": "/psp/paymentorders/{{ page.payment_order_id }}/orderItems" },
+        "metadata": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/metadata" },
+        "payments": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/payments" },
+        "currentPayment": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/currentpayment" }
+    },
+    "operations": [
+        {
+            "method": "PATCH",
+            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}",
+            "rel": "update-paymentorder-abort",
+            "contentType": "application/json"
+        },
+        {
+            "method": "GET",
+            "href": "{{ page.front_end_url }}/paymentmenu/{{ page.payment_token }}",
+            "rel": "redirect-paymentorder",
+            "contentType": "text/html"
+        },
+        {
+            "method": "GET",
+            "href": "{{ page.front_end_url }}/paymentmenu/core/scripts/client/px.paymentmenu.client.js?token={{ page.payment_token }}&culture=nb-NO",
+            "rel": "view-paymentorder",
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+The response given when aborting a payment order is equivalent to a `GET`
+request towards the `paymentorders` resource, as displayed above.
+with its `state` set to `Aborted`.
+
+### Cancel
+
+{% include payment-order-cancel.md %}
+
+#### Reversal
+
+{% include payment-order-reversal.md %}
+
+### Transactions
+
+{% include transactions.md api_resource="paymentorders"
+documentation_section="checkout" %}
+
+#### Transaction
+
+{% include transaction.md api_resource="paymentorders"
+documentation_section="checkout" %}
+
+## Recurring Payments
+
+If you want to enable subsequent recurring – server-to-server – payments, you
+need to create a **recurrence token**. This token will be utilized after the
+initial payment order. **Recurring payments must be activated on the contract
+with Swedbank Pay in order to work.**
+
+### Recurrence Token
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-tabs] Use spaces instead of tabs](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)